src/patternfly/components/Table/table.scss

```
@@ -254,9 +254,25 @@
   --#{$table}--m-sticky-header--ZIndex: calc(var(--pf-t--global--z-index--xs) + 1);
   --#{$table}--m-sticky-header--border--ZIndex: calc(var(--pf-t--global--z-index--xs) + 2);
+  --#{$table}--m-sticky-header--BorderRadius: 0;
+  --#{$table}--m-sticky-header--BoxShadow: 0;
+  --#{$table}--m-sticky-header--BorderBlockEndWidth: var(--#{$table}--border-width--base);
+  --#{$table}--m-sticky-header--BorderBlockEndColor: var(--#{$table}--BorderColor);
+  --#{$table}--m-sticky-header--BorderRadius--glass: var(--pf-t--global--border--radius--medium);
+  --#{$table}--m-sticky-header--BoxShadow--glass: var(--pf-t--global--box-shadow--sm);
+  --#{$table}--m-sticky-header--TransitionTimingFunction--BackgroundColor: var(--pf-t--global--motion--timing-function--default);
+  --#{$table}--m-sticky-header--TransitionDuration--BackgroundColor: var(--pf-t--global--motion--duration--fade--default);
+
+  :where(.pf-v6-theme-glass) & {
```

Stylelint (17.6.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)